### PR TITLE
Added Water Intake Calculator

### LINKED ISSUE #152: Water Intake Calculator
Create a calculator that helps users determine their daily water intake based on factors such as body weight, activity level, and climate.

### PR DESCRIPTION
**Pull request by ShatilKhan**
# Fixes Issue🛠️
Closes #152 

# Description👨‍💻 

## Water Intake Calculator

This is a simple calculator that helps you determine your daily water intake based on your body weight, activity level, and climate.

## How to Use

1. Enter your weight in kilograms.
2. Select your activity level (low, medium, high).
3. Select your climate (cold, hot).
4. Click "Calculate" to get your daily water intake in liters.

## Calculation Method

The calculator uses the following formula to calculate the water intake:

- Base intake: 0.033 liters per kg of body weight
- Activity level: add 0.5 liters for medium, 1 liter for high
- Climate: add 0.5 liters for cold, 1 liter for hot

# Type of change📄

- [X] New feature (non-breaking change which adds functionality)

# How this has been tested✅

I added the inputs of weight, activity & climate, got the right results based on my math function.

# Checklist✅ 

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added demonstration in the form of GIF/video file
- [X] I am an Open Source Contributor

# Screenshots/GIF📷
image

### CODE REVIEW

File: Calculators/Water-Intake-Calculator/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# <p align="center">Weight Convertor</p>
```
Comment (Rakesh9100) [inline Calculators/Water-Intake-Calculator/README.md:1]: Pls change this `Weight Convertor` @ShatilKhan

File: Calculators/Water-Intake-Calculator/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<!-- FILEPATH: /g:/Forked-Repos/CalcDiverse/Calculators/Water-Intake-Calculator/index.html -->
```
Comment (Rakesh9100) [inline Calculators/Water-Intake-Calculator/index.html:1]: Delete this line @ShatilKhan

**Comment by Rakesh9100** [comment]
Pls do handle the NaN value as without entering the weight also, it is giving the result @ShatilKhan 

image

**Comment by ShatilKhan** [comment]
> Pls do handle the NaN value as without entering the weight also, it is giving the result @ShatilKhan
> 
> image

image

**Comment by Rakesh9100** [comment]
Merged!! 🎉 @ShatilKhan

**Pull request merged**